examples/src/main/scala/sttp/tapir/examples/ZioExampleHttp4sServer.scala

```diff
@@ -34,13 +32,16 @@ object ZioExampleHttp4sServer extends App {
   val petEndpoint: Endpoint[Int, String, Pet, Nothing] =
     endpoint.get.in("pet" / path[Int]("petId")).errorOut(stringBody).out(jsonBody[Pet])
 
+  val zioEndpoint: Endpoint[Int, String, Pet, Nothing] =
```

Comment:
maybe we can have just a single endpoint, that is, convert `petEndpoint` to the "ZIO-layer" version? Having two different ones might be confusing to users.

Or maybe, a comment on two possible approaches would be better?

Reply:
Adam I didn't understand your point.
I have 2 EP there - "petEndpoint" and "zioEndpoint". And combine them in a service.

If you don't like this, make your choice and give me an exaple
Thanks for reviewing this

Reply:
I guess my suggestion is to add a comment, which clarifies what's the difference in the implementation of the `petEndpoint` and `zioEndpoint`

Reply:
Updated